shared/src/androidMain/kotlin/io/customer/shared/Platform.kt

```diff
@@ -1,7 +1,12 @@
 package io.customer.shared
 
-class AndroidPlatform : Platform {
+import android.content.Context
+
+/**
+ * Android platform class to hold Android specific properties e.g. [Context], etc.
+ */
+class AndroidPlatform(
+    val applicationContext: Context,
+) : Platform {
```

Comment:
The idea behind this is to utilize digraph for holding platform based properties inside a wrapper, which can be helpful when communicating with mobile OS (e.g. `Context` in Android is often required and was previously in `CustomerIOConfig`).